### DISCUSSION TITLE
Blink LED tutorial using Alchitry Labs 2.0.19

### DISCUSSION THREAD

**Discussion by cdbierl** [alchitry/Alchitry-Labs-V2#37] [Q&A] [answered]
I was able to toggle the LED via button press and verify in the simulator. I was not able to see the simulator blink the LED when I moved to the time based tutorial. Is there a limitation in the simulator that might explain this? 

https://learn.sparkfun.com/tutorials/programming-an-fpga/all#blink-an-led

**Comment by cdbierl** [comment]
This might be my issue: 
![image](https://github.com/user-attachments/assets/eb2df42d-4f3f-40b6-8aff-41af74490f47)

**Comment by alchitry** [comment, marked as answer]
Generally speaking, I'd recommend our tutorials over SparkFun's just because they're more up to date. https://alchitry.com/tutorials/

That tutorial was written for Alchitry Labs V1 and not V2. There are some subtle differences in Lucid V1 vs V2.

It looks like you found it but this is our equivalent tutorial https://alchitry.com/tutorials/synchronous-logic/
